# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/gmeasures/hotfix-autogen.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- gmeasures-0.7.orig/autogen.sh	2007-02-01 14:02:27.000000000 +0100
+++ gmeasures-0.7/autogen.sh	2026-07-26 06:34:51.190033167 +0200
@@ -120,8 +120,8 @@
 
 if test x$NOCONFIGURE = x; then
   echo Running $srcdir/configure $conf_flags "$@" ...
-  $srcdir/configure $conf_flags "$@" \
-  && echo Now type \`make\' to compile $PKG_NAME
+  $srcdir/configure $conf_flags "$@" && \
+  echo Now type \`make\' to compile $PKG_NAME
 else
   echo Skipping configure process.
 fi
